tools/wptserve/tests/functional/test_pipes.py

```diff
@@ -120,7 +121,11 @@ def test_sub_url_base_via_filename_with_query(self):
 
     def test_sub_uuid(self):
         resp = self.request("/sub_uuid.sub.txt")
-        self.assertRegexpMatches(resp.read().rstrip(), b"Before [a-f0-9-]+ After")
```

Comment:
Shouldn't this just use https://six.readthedocs.io/#six.assertRegex then?

Reply:
Yes, I had thought about it. Then I thought maybe for consistence of calling "self.*" through the class?

Reply:
Updated :)